cmd/limactl/save.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"github.com/lima-vm/lima/pkg/instance"
+	networks "github.com/lima-vm/lima/pkg/networks/reconcile"
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/spf13/cobra"
+)
+
+func newSaveCommand() *cobra.Command {
+	saveCmd := &cobra.Command{
+		Use:               "save INSTANCE",
+		Short:             "Save an instance",
```

Comment:
This also stop the instance but the name or the help do not explain this. I think this is confusing user interface even if we improve the help text, and it is better to have:

    limactl stop --save

Using --save and --force should be an error.

This also more consistent with saveOnStop instance config. You either enable save on stop for the instance, or for specific stop of the instance.